src/modules/land/sagas.ts

```diff
@@ -259,8 +276,13 @@ function* handleTransferLandRequest(action: TransferLandRequestAction) {
 function* handleFetchLandRequest(action: FetchLandsRequestAction) {
   const { address } = action.payload
   try {
-    const [land, authorizations]: [Land[], Authorization[]] = yield call(() => manager.fetchLand(address))
-    yield put(fetchLandsSuccess(address, land, authorizations))
+    const isRentalsEnabled: boolean = yield select(getIsRentalsEnabled)
+
+    const rentals: Rental[] = isRentalsEnabled ? yield call(() => rental.fetchTokenIdsByTenant(address)) : []
```

Comment:
Would you mind changing the yield calls to any of the correct formats?
```suggestion
    const rentals: Rental[] = isRentalsEnabled ? yield call([rental, 'fetchTokenIdsByTenant'], address) : []
```
or 
```
  const rentals: Rental[] = isRentalsEnabled ? yield call([rental, rental.fetchTokenIdsByTenant], address) : []
```